common/src/main/java/com/wynntils/mc/utils/ComponentUtils.java

```diff
@@ -6,81 +6,117 @@
 
 import java.util.Arrays;
 import java.util.Optional;
+
 import net.minecraft.ChatFormatting;
-import net.minecraft.network.chat.Component;
-import net.minecraft.network.chat.Style;
-import net.minecraft.network.chat.TextColor;
+import net.minecraft.network.chat.*;
+import org.jetbrains.annotations.Nullable;
+import org.lwjgl.system.CallbackI;
```

Comment:
This is not used. Typo from autocomplete for CallbackInfo?

Reply:
Oops